dev-libs/pigpio/files/pigpio-66-makefile.patch

```diff
@@ -0,0 +1,117 @@
+--- a/Makefile	2018-03-20 19:57:38.812609685 +0100
++++ b/Makefile	2018-03-20 20:26:58.823455489 +0100
+@@ -12,31 +12,31 @@
+ 
+ CFLAGS	+= -O3 -Wall -pthread
+ 
+-LIB1     = libpigpio.so
++LIB1     = libpigpio.so.0
+ OBJ1     = pigpio.o command.o
+ 
+-LIB2     = libpigpiod_if.so
++LIB2     = libpigpiod_if.so.0
+ OBJ2     = pigpiod_if.o command.o
+ 
+-LIB3     = libpigpiod_if2.so
++LIB3     = libpigpiod_if2.so.0
+ OBJ3     = pigpiod_if2.o command.o
+ 
+ LIB      = $(LIB1) $(LIB2) $(LIB3)
+ 
+-ALL     = $(LIB) x_pigpio x_pigpiod_if x_pigpiod_if2 pig2vcd pigpiod pigs
++ALL     = $(LIB) pig2vcd pigpiod pigs
+ 
+-LL1      = -L. -lpigpio -pthread -lrt
++LL1      = -L. $(LIB1) -pthread -lrt
+ 
+-LL2      = -L. -lpigpiod_if -pthread -lrt
++LL2      = -L. $(LIB2) -pthread -lrt
+ 
+-LL3      = -L. -lpigpiod_if2 -pthread -lrt
++LL3      = -L. $(LIB3) -pthread -lrt
+ 
+-prefix = /usr/local
++prefix = /usr
```

Comment:
Why are you patching the prefix instead of passing it to `emake`? Avoid unnecessary patching, especially if it's non-upstreamable.